truss/templates/server/model_wrapper.py

```diff
@@ -162,6 +168,21 @@ def try_load(self):
                 gap_seconds=1.0,
             )
 
+    def set_truss_schema(self):
+        parameters = (
```

Comment:
curious: what would happen for the following truss
```
def preprocess(self, a: Type1) -> Type2:
def predict(self, a):
     pass
```
As far as I can tell, the input should be Type1 and output should be Type2, but I don't think the output type would be reflected here. but that's probably okay? Just thinking through all the cases.